Fixed dropwizard and netty test issues

prydin opened the pull request:
Fixed #58 (Dropwizard fails when port 9091 is used)

Fixed #59 (netty test fails when connecting over ipv6)

trask commented on java-agent/instrumentation/netty-4.0/src/test/groovy/Netty40ClientTest.groovy:
```diff
@@ -61,7 +61,7 @@ class Netty40ClientTest extends HttpClientTest<NettyHttpClientDecorator> {
   def "connection error (unopened port)"() {
     given:
-    def uri = new URI("http://localhost:$UNUSABLE_PORT/")
+    def uri = new URI("http://127.0.0.1:$UNUSABLE_PORT/") // Use numeric address to avoid ipv4/ipv6 confusion
```
👍

trask commented on java-agent/instrumentation/netty-4.0/src/test/groovy/Netty40ClientTest.groovy:
```diff
@@ -71,6 +71,7 @@ class Netty40ClientTest extends HttpClientTest<NettyHttpClientDecorator> {
     then:
     def ex = thrown(Exception)
     def thrownException = ex instanceof ExecutionException ? ex.cause : ex
+    println "Exception: $ex"
```
Just checking if this was an artifact of debugging, or intentionally part of PR, I'm ok either way

prydin replied:
This is leftover debugging. I'll get rid of it.

trask commented on java-agent/instrumentation/dropwizard/src/test/groovy/DropwizardTest.groovy:
```diff
@@ -29,9 +30,11 @@ class DropwizardTest extends HttpServerTest<DropwizardTestSupport, Servlet3Decor
       null,
-      ConfigOverride.config("server.applicationConnectors[0].port", "$port"))
+      ConfigOverride.config("server.applicationConnectors[0].port", "$port"),
+      ConfigOverride.config("server.adminConnectors[0].port", "" + PortUtils.randomOpenPort()))
```
The `check` build failed, flagging this line, see https://829-210933087-gh.circle-artifacts.com/0/home/circleci/dd-trace-java/reports/java-agent/instrumentation/dropwizard/codenarc/test.html

prydin replied:
OK. That's a lazy way of making a string from an int. Not that it matters in this context, but we want clean builds, so I'll change it!

trask commented:
@prydin, everything looks good! Can you rebase this PR and target master now that #55 is merged?

prydin commented:
Will do as soon as I’m back to my laptop.

prydin commented:
That ended up being a really messy merge... :( Lots of rename and add conflicts. I may just isolate my changes and issue a new PR against master instead.

```
CONFLICT (rename/rename): Rename "dd-java-agent/src/test/resources/logback.xml"->"java-agent/instrumentation/spring-webmvc-3.1/src/test/resources/logback.xml" in branch "HEAD" rename "dd-java-agent/src/test/resources/logback.xml"->"java-agent/src/test/resources/logback.xml" in "upstream/master"
Auto-merging java-agent/instrumentation/rxjava-1/src/main/java/io/opentelemetry/auto/instrumentation/rxjava/TracedSubscriber.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/rxjava-1/src/main/java/io/opentelemetry/auto/instrumentation/rxjava/TracedSubscriber.java
Auto-merging java-agent/instrumentation/rxjava-1/src/main/java/io/opentelemetry/auto/instrumentation/rxjava/TracedOnSubscribe.java
CONFLICT (add/add): Merge conflict in java-agent/instrumentation/rxjava-1/src/main/java/io/opentelemetry/auto/instrumentation/rxjava/TracedOnSubscribe.java
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/play-ws-2.1/src/test/groovy/PlayWSClientTest.groovy"->"java-agent/instrumentation/play-ws-2.1/src/test/groovy/PlayWSClientTest.groovy" in branch "HEAD" rename "dd-java-agent/instrumentation/play-ws-2.1/src/test/groovy/PlayWSClientTest.groovy"->"java-agent/instrumentation/play-ws-1/src/test/groovy/PlayWSClientTest.groovy" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/play-ws-2/src/main/java/datadog/trace/instrumentation/playws2/PlayWSClientInstrumentation.java"->"java-agent/instrumentation/play-ws-2.1/src/main/java/io/opentelemetry/auto/instrumentation/playws21/PlayWSClientInstrumentation.java" in branch "HEAD" rename "dd-java-agent/instrumentation/play-ws-2/src/main/java/datadog/trace/instrumentation/playws2/PlayWSClientInstrumentation.java"->"java-agent/instrumentation/play-ws-1/src/main/java/io/opentelemetry/auto/instrumentation/playws1/PlayWSClientInstrumentation.java" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/play-ws-2/src/test/groovy/PlayWSClientTest.groovy"->"java-agent/instrumentation/play-ws-1/src/test/groovy/PlayWSClientTest.groovy" in branch "HEAD" rename "dd-java-agent/instrumentation/play-ws-2/src/test/groovy/PlayWSClientTest.groovy"->"java-agent/instrumentation/play-ws-2/src/test/groovy/PlayWSClientTest.groovy" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/play-ws-2/src/main/java/datadog/trace/instrumentation/playws2/PlayWSClientDecorator.java"->"java-agent/instrumentation/play-ws-1/src/main/java/io/opentelemetry/auto/instrumentation/playws1/PlayWSClientDecorator.java" in branch "HEAD" rename "dd-java-agent/instrumentation/play-ws-2/src/main/java/datadog/trace/instrumentation/playws2/PlayWSClientDecorator.java"->"java-agent/instrumentation/play-ws-2.1/src/main/java/io/opentelemetry/auto/instrumentation/playws21/PlayWSClientDecorator.java" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/play-ws-2/src/main/java/datadog/trace/instrumentation/playws2/HeadersInjectAdapter.java"->"java-agent/instrumentation/play-ws-1/src/main/java/io/opentelemetry/auto/instrumentation/playws1/HeadersInjectAdapter.java" in branch "HEAD" rename "dd-java-agent/instrumentation/play-ws-2/src/main/java/datadog/trace/instrumentation/playws2/HeadersInjectAdapter.java"->"java-agent/instrumentation/play-ws-2/src/main/java/io/opentelemetry/auto/instrumentation/playws2/HeadersInjectAdapter.java" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/play-2.6/src/main/java/datadog/trace/instrumentation/play26/PlayInstrumentation.java"->"java-agent/instrumentation/play-2.4/src/main/java/io/opentelemetry/auto/instrumentation/play24/PlayInstrumentation.java" in branch "HEAD" rename "dd-java-agent/instrumentation/play-2.6/src/main/java/datadog/trace/instrumentation/play26/PlayInstrumentation.java"->"java-agent/instrumentation/play-2.6/src/main/java/io/opentelemetry/auto/instrumentation/play26/PlayInstrumentation.java" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/netty-4.1/src/main/java/datadog/trace/instrumentation/netty41/client/HttpClientTracingHandler.java"->"java-agent/instrumentation/netty-4.1/src/main/java/io/opentelemetry/auto/instrumentation/netty41/client/HttpClientTracingHandler.java" in branch "HEAD" rename "dd-java-agent/instrumentation/netty-4.1/src/main/java/datadog/trace/instrumentation/netty41/client/HttpClientTracingHandler.java"->"java-agent/instrumentation/netty-4.0/src/main/java/io/opentelemetry/auto/instrumentation/netty40/client/HttpClientTracingHandler.java" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/netty-4.1/src/main/java/datadog/trace/instrumentation/netty41/client/HttpClientRequestTracingHandler.java"->"java-agent/instrumentation/netty-4.1/src/main/java/io/opentelemetry/auto/instrumentation/netty41/client/HttpClientRequestTracingHandler.java" in branch "HEAD" rename "dd-java-agent/instrumentation/netty-4.1/src/main/java/datadog/trace/instrumentation/netty41/client/HttpClientRequestTracingHandler.java"->"java-agent/instrumentation/netty-4.0/src/main/java/io/opentelemetry/auto/instrumentation/netty40/client/HttpClientRequestTracingHandler.java" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/netty-4.1/src/test/groovy/NettyServerTestInstrumentation.java"->"java-agent/instrumentation/netty-4.0/src/test/groovy/NettyServerTestInstrumentation.java" in branch "HEAD" rename "dd-java-agent/instrumentation/netty-4.1/src/test/groovy/NettyServerTestInstrumentation.java"->"java-agent/instrumentation/netty-4.1/src/test/groovy/NettyServerTestInstrumentation.java" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/netty-4.1/src/main/java/datadog/trace/instrumentation/netty41/server/NettyRequestExtractAdapter.java"->"java-agent/instrumentation/netty-4.0/src/main/java/io/opentelemetry/auto/instrumentation/netty40/server/NettyRequestExtractAdapter.java" in branch "HEAD" rename "dd-java-agent/instrumentation/netty-4.1/src/main/java/datadog/trace/instrumentation/netty41/server/NettyRequestExtractAdapter.java"->"java-agent/instrumentation/netty-4.1/src/main/java/io/opentelemetry/auto/instrumentation/netty41/server/NettyRequestExtractAdapter.java" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/netty-4.1/src/main/java/datadog/trace/instrumentation/netty41/server/HttpServerTracingHandler.java"->"java-agent/instrumentation/netty-4.0/src/main/java/io/opentelemetry/auto/instrumentation/netty40/server/HttpServerTracingHandler.java" in branch "HEAD" rename "dd-java-agent/instrumentation/netty-4.1/src/main/java/datadog/trace/instrumentation/netty41/server/HttpServerTracingHandler.java"->"java-agent/instrumentation/netty-4.1/src/main/java/io/opentelemetry/auto/instrumentation/netty41/server/HttpServerTracingHandler.java" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/netty-4.1/src/main/java/datadog/trace/instrumentation/netty41/ChannelFutureListenerInstrumentation.java"->"java-agent/instrumentation/netty-4.0/src/main/java/io/opentelemetry/auto/instrumentation/netty40/ChannelFutureListenerInstrumentation.java" in branch "HEAD" rename "dd-java-agent/instrumentation/netty-4.1/src/main/java/datadog/trace/instrumentation/netty41/ChannelFutureListenerInstrumentation.java"->"java-agent/instrumentation/netty-4.1/src/main/java/io/opentelemetry/auto/instrumentation/netty41/ChannelFutureListenerInstrumentation.java" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/hibernate/core-4.3/src/test/java/Value.java"->"java-agent/instrumentation/hibernate/core-4.3/src/test/java/Value.java" in branch "HEAD" rename "dd-java-agent/instrumentation/hibernate/core-4.3/src/test/java/Value.java"->"java-agent/instrumentation/hibernate/core-4.0/src/test/java/Value.java" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/hibernate/core-4.0/src/test/groovy/QueryTest.groovy"->"java-agent/instrumentation/hibernate/core-4.0/src/test/groovy/QueryTest.groovy" in branch "HEAD" rename "dd-java-agent/instrumentation/hibernate/core-4.0/src/test/groovy/QueryTest.groovy"->"java-agent/instrumentation/hibernate/core-3.3/src/test/groovy/QueryTest.groovy" in "upstream/master"
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/hibernate/core-4.0/src/test/groovy/CriteriaTest.groovy"->"java-agent/instrumentation/hibernate/core-4.0/src/test/groovy/CriteriaTest.groovy" in branch "HEAD" rename "dd-java-agent/instrumentation/hibernate/core-4.0/src/test/groovy/CriteriaTest.groovy"->"java-agent/instrumentation/hibernate/core-3.3/src/test/groovy/CriteriaTest.groovy" in "upstream/master"
Auto-merging java-agent/instrumentation/google-http-client/src/main/java/io/opentelemetry/auto/instrumentation/googlehttpclient/GoogleHttpClientInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/google-http-client/src/main/java/io/opentelemetry/auto/instrumentation/googlehttpclient/GoogleHttpClientInstrumentation.java
Auto-merging java-agent/instrumentation/google-http-client/src/main/java/io/opentelemetry/auto/instrumentation/googlehttpclient/GoogleHttpClientDecorator.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/google-http-client/src/main/java/io/opentelemetry/auto/instrumentation/googlehttpclient/GoogleHttpClientDecorator.java
Auto-merging java-agent/instrumentation/elasticsearch/transport-6/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch6/Elasticsearch6TransportClientInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/elasticsearch/transport-6/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch6/Elasticsearch6TransportClientInstrumentation.java
Auto-merging java-agent/instrumentation/elasticsearch/transport-5/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch5/Elasticsearch5TransportClientInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/elasticsearch/transport-5/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch5/Elasticsearch5TransportClientInstrumentation.java
Auto-merging java-agent/instrumentation/elasticsearch/transport-5.3/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch5_3/Elasticsearch53TransportClientInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/elasticsearch/transport-5.3/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch5_3/Elasticsearch53TransportClientInstrumentation.java
Auto-merging java-agent/instrumentation/elasticsearch/transport-2/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch2/Elasticsearch2TransportClientInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/elasticsearch/transport-2/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch2/Elasticsearch2TransportClientInstrumentation.java
Auto-merging java-agent/instrumentation/elasticsearch/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch/ElasticsearchTransportClientDecorator.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/elasticsearch/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch/ElasticsearchTransportClientDecorator.java
Auto-merging java-agent/instrumentation/elasticsearch/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch/ElasticsearchRestClientDecorator.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/elasticsearch/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch/ElasticsearchRestClientDecorator.java
Auto-merging java-agent/instrumentation/elasticsearch/rest-6.4/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch6_4/Elasticsearch6RestClientInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/elasticsearch/rest-6.4/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch6_4/Elasticsearch6RestClientInstrumentation.java
Auto-merging java-agent/instrumentation/elasticsearch/rest-5/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch5/Elasticsearch5RestClientInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/elasticsearch/rest-5/src/main/java/io/opentelemetry/auto/instrumentation/elasticsearch5/Elasticsearch5RestClientInstrumentation.java
CONFLICT (rename/rename): Rename "dd-java-agent/instrumentation/jax-rs-annotations-1/src/test/groovy/JettyTestInstrumentation.java"->"java-agent/instrumentation/dropwizard/src/test/groovy/JettyTestInstrumentation.java" in branch "HEAD" rename "dd-java-agent/instrumentation/jax-rs-annotations-1/src/test/groovy/JettyTestInstrumentation.java"->"java-agent/instrumentation/jax-rs-annotations-1/src/test/groovy/JettyTestInstrumentation.java" in "upstream/master"
Auto-merging java-agent/instrumentation/dropwizard/src/test/groovy/DropwizardTest.groovy
CONFLICT (content): Merge conflict in java-agent/instrumentation/dropwizard/src/test/groovy/DropwizardTest.groovy
Auto-merging java-agent/instrumentation/dropwizard/dropwizard-views/src/main/java/io/opentelemetry/auto/instrumentation/dropwizard/view/DropwizardViewInstrumentation.java
Auto-merging java-agent/instrumentation/datastax-cassandra-3/src/main/java/io/opentelemetry/auto/instrumentation/datastax/cassandra/TracingSession.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/datastax-cassandra-3/src/main/java/io/opentelemetry/auto/instrumentation/datastax/cassandra/TracingSession.java
Auto-merging java-agent/instrumentation/couchbase-2.6/src/main/java/io/opentelemetry/auto/instrumentation/couchbase/client/CouchbaseCoreInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/couchbase-2.6/src/main/java/io/opentelemetry/auto/instrumentation/couchbase/client/CouchbaseCoreInstrumentation.java
Auto-merging java-agent/instrumentation/aws-java-sdk-1.11.0/src/main/java/io/opentelemetry/auto/instrumentation/aws/v0/TracingRequestHandler.java
CONFLICT (add/add): Merge conflict in java-agent/instrumentation/aws-java-sdk-1.11.0/src/main/java/io/opentelemetry/auto/instrumentation/aws/v0/TracingRequestHandler.java
Auto-merging java-agent/instrumentation/apache-httpclient-4/src/main/java/io/opentelemetry/auto/instrumentation/apachehttpclient/ApacheHttpClientInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/apache-httpclient-4/src/main/java/io/opentelemetry/auto/instrumentation/apachehttpclient/ApacheHttpClientInstrumentation.java
Auto-merging java-agent/instrumentation/apache-httpclient-4/src/main/java/io/opentelemetry/auto/instrumentation/apachehttpclient/ApacheHttpClientDecorator.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/apache-httpclient-4/src/main/java/io/opentelemetry/auto/instrumentation/apachehttpclient/ApacheHttpClientDecorator.java
Auto-merging java-agent/instrumentation/apache-httpasyncclient-4/src/main/java/io/opentelemetry/auto/instrumentation/apachehttpasyncclient/ApacheHttpAsyncClientInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/apache-httpasyncclient-4/src/main/java/io/opentelemetry/auto/instrumentation/apachehttpasyncclient/ApacheHttpAsyncClientInstrumentation.java
Auto-merging java-agent/instrumentation/apache-httpasyncclient-4/src/main/java/io/opentelemetry/auto/instrumentation/apachehttpasyncclient/ApacheHttpAsyncClientDecorator.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/apache-httpasyncclient-4/src/main/java/io/opentelemetry/auto/instrumentation/apachehttpasyncclient/ApacheHttpAsyncClientDecorator.java
Auto-merging java-agent/instrumentation/akka-http-10.0/src/main/java/io/opentelemetry/auto/instrumentation/akkahttp/AkkaHttpServerInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/akka-http-10.0/src/main/java/io/opentelemetry/auto/instrumentation/akkahttp/AkkaHttpServerInstrumentation.java
Auto-merging java-agent/instrumentation/akka-http-10.0/src/main/java/io/opentelemetry/auto/instrumentation/akkahttp/AkkaHttpServerDecorator.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/akka-http-10.0/src/main/java/io/opentelemetry/auto/instrumentation/akkahttp/AkkaHttpServerDecorator.java
Auto-merging java-agent/instrumentation/akka-http-10.0/src/main/java/io/opentelemetry/auto/instrumentation/akkahttp/AkkaHttpClientInstrumentation.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/akka-http-10.0/src/main/java/io/opentelemetry/auto/instrumentation/akkahttp/AkkaHttpClientInstrumentation.java
Auto-merging java-agent/instrumentation/akka-http-10.0/src/main/java/io/opentelemetry/auto/instrumentation/akkahttp/AkkaHttpClientDecorator.java
CONFLICT (content): Merge conflict in java-agent/instrumentation/akka-http-10.0/src/main/java/io/opentelemetry/auto/instrumentation/akkahttp/AkkaHttpClientDecorator.java
Auto-merging java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/tooling/AgentTracerImpl.java
Auto-merging java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/ServerDecorator.java
CONFLICT (add/add): Merge conflict in java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/ServerDecorator.java
CONFLICT (rename/delete): dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/decorator/OrmClientDecorator.java deleted in HEAD and renamed to java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/OrmClientDecorator.java in upstream/master. Version upstream/master of java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/OrmClientDecorator.java left in tree.
CONFLICT (rename/delete): dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/decorator/HttpServerDecorator.java deleted in HEAD and renamed to java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/HttpServerDecorator.java in upstream/master. Version upstream/master of java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/HttpServerDecorator.java left in tree.
CONFLICT (rename/delete): dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/decorator/HttpClientDecorator.java deleted in HEAD and renamed to java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/HttpClientDecorator.java in upstream/master. Version upstream/master of java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/HttpClientDecorator.java left in tree.
Auto-merging java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/DatabaseClientDecorator.java
CONFLICT (add/add): Merge conflict in java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/DatabaseClientDecorator.java
Auto-merging java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/ClientDecorator.java
CONFLICT (add/add): Merge conflict in java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/ClientDecorator.java
CONFLICT (rename/delete): dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/decorator/BaseDecorator.java deleted in HEAD and renamed to java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/BaseDecorator.java in upstream/master. Version upstream/master of java-agent/agent-tooling/src/main/java/io/opentelemetry/auto/decorator/BaseDecorator.java left in tree.
Auto-merging java-agent/agent-bootstrap/src/main/java/io/opentelemetry/auto/instrumentation/api/AgentTracer.java
CONFLICT (content): Merge conflict in java-agent/agent-bootstrap/src/main/java/io/opentelemetry/auto/instrumentation/api/AgentTracer.java
Auto-merging java-agent/agent-bootstrap/src/main/java/io/opentelemetry/auto/instrumentation/api/AgentSpan.java
CONFLICT (add/add): Merge conflict in java-agent/agent-bootstrap/src/main/java/io/opentelemetry/auto/instrumentation/api/AgentSpan.java
Auto-merging buildSrc/src/main/groovy/MuzzlePlugin.groovy
```

trask commented:
Sounds good
